02_Util/src/util/StringOperations.ts

```diff
@@ -0,0 +1,6 @@
+export function splitOnce(value: string, separator: string): [string, string?] {
```

Comment:
This seems to duplicate `WebsocketNetworkConnection:306` can we remove the private method then?